packages/react-dom/src/__tests__/ReactDOMServerPartialHydration-test.internal.js

```diff
@@ -208,9 +208,17 @@ describe('ReactDOMServerPartialHydration', () => {
     // On the client we don't have all data yet but we want to start
     // hydrating anyway.
     suspend = true;
-    ReactDOM.hydrateRoot(container, <App />);
+    ReactDOM.hydrateRoot(container, <App />, {
+      onRecoverableError(error) {
+        Scheduler.unstable_yieldValue(error.message);
+      },
+    });
     if (gate(flags => flags.enableClientRenderFallbackOnHydrationMismatch)) {
-      Scheduler.unstable_flushAll();
+      // Hydration error is logged
+      expect(Scheduler).toFlushAndYield([
+        'An error occurred during hydration. The server HTML was replaced ' +
```

Comment:
This is actually not a real error. What happens is that since `<Child>` suspended the hydration pointer is off for the remainder where we "attempt to hydrate" (Really we're just "warming up"  the components instead of bailing) so everything after a component suspends leads to a hydration mismatch.